.github/workflows/ci.yml

```diff
@@ -29,10 +29,12 @@ jobs:
     strategy:
       matrix:
         python-version: ["3.8", "3.9", "3.10", "3.11", "3.12"]
-        django-version: [22, 32, 40, 41, 42]
+        django-version: [42, 50]
```

Comment:
Do we need to drop support for Django 3? I feel like many people might still be using it.

Reply:
Good morning! Well, good question. I always go with: I dont support whats not being supported. Especially, since it doesn't mean, it's not working, we just dont guarentee it.

But I'd say, let's agree on a strategy. What would be best in your opinion?

Reply:
I like supporting one version behind if it's not too much of a hassle, if it runs in CI it's easier not to break it.

But you do make a good point. I think we could drop Django 3 if it becomes difficult to support it moving forward, e.g. with a future feature or enhancement.

Reply:
@pcraciunoiu I re-added Django 3.2